website/content/partials/known-issues/ephemeral-loggers-memory-leak.mdx

```diff
@@ -1,14 +1,39 @@
 ### Vault is storing references to ephemeral sub-loggers leading to a memory leak
 
-Vault is unexpectedly storing references to ephemeral sub-loggers which prevents them from being cleaned up, leading to
-a memory leak. This impacts many areas of Vault, but primarily logins in Enterprise.
+In Vault versions predating 1.13.7, 1.14.3, and 1.15.0 Vault does not honor a
+modified `log_level` configuration for certain subsystem loggers on SIGHUP.
+
+This is known to affect the `resolver.watcher` and `replication.index.*`
+sub-loggers.
+
+This can be seen after modifying the `log_level` and issuing a reload (SIGHUP),
+For example, after starting a server with `log_level: "trace"` and modifying it
+to `log_level: "info"` the following lines appear after reload:
+
+```
+[TRACE] resolver.watcher: dr mode doesn't have failover support, returning
+...
+[DEBUG] replication.index.perf: saved checkpoint: num_dirty=5
+[DEBUG] replication.index.local: saved checkpoint: num_dirty=0
+[DEBUG] replication.index.periodic: starting WAL GC: from=2531280 to=2531280 last=2531536
+```
+
+A change was introduced to resolve this issue:
+`core: All sub-loggers now reflect configured log level on reload. [GH-22038]`
+
+As a result of this change Vault would unexpectedly store references to ephemeral sub-loggers which prevents them from
+being cleaned up, leading to a memory leak. This impacts many areas of Vault, but primarily logins in Enterprise.
 There is no workaround.
 
+The change that introduced this leak will be reverted in the next release, restoring the previously known behavior of
+sub-loggers not updating log levels on SIGHUP.
+
 #### Affected versions
 
 This issue affects Vault Community and Enterprise versions:
+
 - 1.13.7+
 - 1.14.3+
 - 1.15.0+
```

Comment:
This might end up being confusing to some readers, since the change will be reverted in 1.13.10, etc. Maybe we should use specific release versions instead?

```
>  for tags in 9a9f21ff67c3082c9eccccd3eb444ed92c26d9df 4c8cc87794ed2d989f515cd30c1c1b953d092ef3 4c0edc73b2f9e6e0e2e8b8def39187fb5740cabe; git tag --contains $tags | grep "^v" | grep -v "ent"; end
v1.13.7
v1.13.8
v1.13.9
v1.15.0
v1.15.0-rc1
v1.15.1
v1.14.3
v1.14.4
v1.14.5
```

Reply:
You can say something like 1.13.7-1.13.9, 1.14...,1.15... and give ranges.

Reply:
This has been split into its own known issue so not relevant to this PR anymore.